projects/user-telemetry-client/tsconfig.json

```diff
@@ -0,0 +1,69 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```

Comment:
VSCode gives us excellent intellisense for compiler options – we can probably lose the comments.

Reply:
I said it was early :P